datasets/klue/klue.py

```diff
@@ -0,0 +1,545 @@
+        self.url = url
+        self.file_map = file_map
+
+
+class Klue(datasets.GeneratorBasedBuilder):
+    """The General Language Understanding Evaluation (GLUE) benchmark."""
+
+    BUILDER_CONFIGS = [
+        KlueConfig(
+            name="ynat",
+            features={
+                "guid": datasets.Value("string"),
+                "title": datasets.Value("string"),
+                "predefined_news_category": datasets.Value("string"),
+                "label": datasets.features.ClassLabel(names=["IT과학", "경제", "사회", "생활문화", "세계", "스포츠", "정치"]),
+                "annotations": {
+                    "annotators": [datasets.Value("string")],
+                    "annotations": {
+                        "first-scope": [datasets.Value("string")],
+                        "second-scope": [datasets.Value("string")],
+                        "third-scope": [datasets.Value("string")],
+                    },
+                },
+                "url": datasets.Value("string"),
+                "date": datasets.Value("string"),
+            },
+            description=textwrap.dedent(
+                """\
+            In topic classification (TC), the goal is to predict the topic of a given text
+            snippet. We include TC in our KLUE benchmark, as inferring the topic of a text is a key
+            capability that should be possessed by a language understanding system.
+            Following a typical single  sentence classification task, we introduce YNAT, a Younhap
+            News Agency news headlines for Topic Classification. For Korean, no dataset has been
+            proposed for this task, which motivates us to construct the first Korean topic
+            classification benchmark. In this task, given a news headline, a text classifier must
+            predict a topic which is one of politics, economy, society, culture, world, IT/science,
+            and sports. Macro-F1 score is used to evaluate a system."""
+            ),
+            data_url=_DATA_URLs["ynat"],
+            url=_DESCRIPTION_URLs["ynat"],
+            file_map={
+                "train": "ynat-v1_train.json",
+                "dev": "ynat-v1_dev.json",
+            },
+        ),
+        KlueConfig(
+            name="sts",
+            features={
+                "guid": datasets.Value("string"),
+                "source": datasets.Value("string"),
+                "sentence1": datasets.Value("string"),
+                "sentence2": datasets.Value("string"),
+                "labels": {
+                    "label": datasets.Value("float64"),
+                    "real-label": datasets.Value("float64"),
+                    "binary-label": datasets.ClassLabel(names=["negative", "positive"]),
+                },
+                "annotations": {
+                    "agreement": datasets.Value("string"),
+                    "annotators": [datasets.Value("string")],
+                    "annotations": [datasets.Value("int32")],
+                },
```

Comment:
```suggestion
                "guid": datasets.Value("string"),
                "source": datasets.Value("string"),
                "sentence1": datasets.Value("string"),
                "sentence2": datasets.Value("string"),
                "labels": {
                    "label": datasets.Value("float64"),
                    "real-label": datasets.Value("float64"),
                    "binary-label": datasets.ClassLabel(names=["negative", "positive"]),
                },
```
I think they don't need to be added because they just show how curators decide the gold label and they can give confusion to users.